mmv1/templates/terraform/examples/network_services_gateway_secure_web_proxy.tf.tmpl

```diff
@@ -51,6 +51,7 @@ resource "google_network_services_gateway" "{{$.PrimaryResourceId}}" {
   location                             = "us-central1"
   addresses                            = ["10.128.0.99"]
   type                                 = "SECURE_WEB_GATEWAY"
+  routing_mode                         = "EXPLICIT_ROUTING_MODE"
```

Comment:
Do you mind removing all of the `EXPLICIT_ROUTING_MODE` in the tests?

Reply:
Done